### PR TITLE
Generate ensemble members with a Sobol sequence

### PR DESCRIPTION
**Pull request by rplzzz**
This change allows us to generate an arbitrary number of parameter combinations, distributed over the parameter space with uniform density.

### CODE REVIEW

File: R/main.R [hunk 1]
```diff
@@ -1,118 +1,149 @@
...
+  invisible(scenObjects)
+}
+
+#' Generate the ensemble members for a single set of parameters
+#'
+#' This generates one each of the Perfect, Lagged, and Linear scenario types
+#' using the input parameters.  The return value is a list of the three
+#' ScenarioInfo objects for the scenarios generated.
+#'
+#' @param agFor The logit exponent the ag/forest(?) nest
```
Comment (kvcalvin) [inline R/main.R]: Yes, this is the logit exponent under the AgroForest nest

File: R/main.R [hunk 2]
```diff
@@ -1,118 +1,149 @@
...
+#' @param agForNonPast The logit exponent for the non-pasture(?) nest
```
Comment (kvcalvin) [inline R/main.R]: Yes, this is for the AgForest_NonPasture nest (essentially, the competition between crop, grass/shrub, and forest

File: R/main.R (hunk 1 above)
Comment (kvcalvin) [reply R/main.R]: Essentially the competition between pasture and all other arable land

File: R/main.R [hunk 3]
```diff
@@ -1,118 +1,149 @@
...
+                          aLogitUseDefault = FALSE,
+                          aLogitAgroForest = agFor,
+                          aLogitAgroForest_NonPasture = agForNonPast,
+                          aLogitCropland = crop,
+                          aScenarioName = scenName,
+                          aFileName = sprintf("Lag_%04d", serialnum),
+                          aOutputDir = aOutputDir)
+
+
+  # Loop over all LINYEARS options and run Linear
```
Comment (kvcalvin) [inline R/main.R]: Is this the right comment for this one?

Comment (rplzzz) [reply R/main.R]: I don't think so.  It looks like I forgot to update it when I reorganized this section.

**Comment by rplzzz** [comment]
Comments and documentation updated as requested.

**Pull request merged**